_i18n/de/resources/user-guides/verification-allos-advanced.md

```diff
@@ -126,57 +126,58 @@ gpg:          There is no indication that the signature belongs to the owner.
 Primary key fingerprint: 81AC 591F E9C4 B65C 5806  AFC3 F0AF 4D46 2A0B DF92
 ```
 
-If your output shows **Good signature**, as in the example, then you may proceed.
+Zeigt dein Output wie im Beispiel **Good signature** an, kannst du fortfahren.
 
-If you see **BAD signature** in the output, **DO NOT CONTINUE.** Instead delete the file `hashes.txt` and go back to [section 3.1](#31-get-hash-file).
+Wenn es **BAD signature** ausgibt, **FAHRE NICHT FORT**. Lösche die `hashes.txt`-Datei und gehe zurück zum [Abschnitt 3.1](#31-hash-datei-herunterladen).
 
-## 4. Download and Verify Binary
+## 4. Download und Verifikation der Binärdatei
 
-This section will cover downloading the Monero binary for your operating system, getting the `SHA256` hash of your download, and verifying that it is correct.
+Dieser Abschnitt behandelt den Download der Monero-Binärdatei für dein Betriebssystem, den Erhalt des `SHA256`-Hashs deines Downloads und dessen Echtheitsverifizierung.
 
-### 4.1. Get Monero binary
+### 4.1. Monero-Binärdatei herunterladen
 
-On Windows or Mac, go to [getmonero.org]({{ site.baseurl_root }}/downloads/) and download the correct file for your operating system. Save the file to your home directory. **Do not extract the files yet.**
+Auf Windows oder Mac: Besuche die [getmonero.org-Seite]({{ site.baseurl_root }}/downloads/) und lade die für dein Betriebssystem passende Datei herunter. Speichere diese Datei in deinem "Home"-Verzeichnis. **Extrahiere die Dateien noch nicht.**
```

Comment:
In some other places you replaced baseurl_root with baseurl.. not sure which is correct.